keep-runtime/build.rs

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
-
+fn main() {}
+/*
```

Comment:
I don't think this affects the runtime behavior - is there any reason to disable it?

Reply:
I don't think any of it is used anymore.